### PR TITLE
allow user to select resolutions

### PR DESCRIPTION
**Pull request by FernetMenta**
see title

maybe we can sort the list later

### CODE REVIEW

File: addons/skin.estuary/xml/Custom_1101_SettingsList.xml [hunk 1]
```diff
@@ -66,6 +66,13 @@
 						<onclick>PlayerProgramSelect</onclick>
 						<visible>Player.HasPrograms</visible>
 					</control>
+					<control type="button" id="11106">
```
Comment (Rechi) [inline addons/skin.estuary/xml/Custom_1101_SettingsList.xml]: needs a different id, this one is already used for subtitle settings

**Comment by FernetMenta** [comment]
jenkins build this please

**Comment by MilhouseVH** [comment]
With the Set Resolution I've found an issue.

My computer monitor has three refresh rates: 50Hz, 59.94Hz, and 60Hz (all at 1920x1080)

s1

All three are enabled in my whitelist (they're the only three that are enabled).

My DESKTOP resolution is 1920x1080 @ 50Hz.

If I watch a 24fps video, it starts in 1920x1080@50Hz.

I can switch refresh rate to 59.94Hz that works - the monitor changes refresh rate. I can change it back to 50Hz, that also works.

But I cannot select 60Hz - doing so has no effect. If I re-open the Set Resolution dialog after selecting 60Hz it shows the previous refresh rate (50Hz or 59.94Hz), so the request to change to 60Hz appears to be completely ignored.

This is a log showing a video starting in 50Hz, and I've changed the refresh rate to 60Hz (which is ignored): http://ix.io/1aCe

Reproducible on RPi and x86_64.

File: xbmc/video/PlayerController.cpp [hunk 2]
```diff
@@ -455,6 +456,38 @@ bool CPlayerController::OnAction(const CAction &action)
...
+            dialog->Add(info.strMode);
+            if (res == currentRes)
+              current = idx;
+            idx++;
+          }
+          dialog->SetHeading(CVariant{g_localizeStrings.Get(39110)});
+          dialog->SetSelected(current);
+          dialog->Open();
+          idx = dialog->GetSelectedItem();
+          if (idx > 0)
```
Comment (FernetMenta) [inline xbmc/video/PlayerController.cpp]: @MilhouseVH I think this should be idx >= 0 or index 0 (valid) won't get selected

**Comment by MilhouseVH** [comment]
s1

Selecting the 60.00 item has no effect, the other two are working. In the log I switch to 75Hz (works), then 60Hz (doesn't do anything).

http://ix.io/1aDJ

File: xbmc/video/PlayerController.cpp (hunk 2 above)
Comment (MilhouseVH) [reply xbmc/video/PlayerController.cpp]: Yes, `>=` fixes the issue - all resolutions and refresh rates listed in the "Select Resolution" dialog are now functioning.

**Pull request merged**